tests/core/_orchestrator/test_orchestrator__submit.py

```diff
@@ -184,22 +189,22 @@ def test_submit_scenario_development_mode_blocked_jobs():
     assert job_3.submit_entity_id == scenario.id
     assert job_3.creation_date == s_time
     assert job_3.stacktrace == []
-    assert len(job_3._subscribers) == 2  # submission._update_submission_status and orchestrator._on_status_change
-    assert job_3._subscribers[0].__code__ == Submission._update_submission_status.__code__
+    assert len(job_3._subscribers) == 2  # _Orchestrator._update_submission_status and orchestrator._on_status_change
+    assert job_3._subscribers[0].__code__ == _Orchestrator._update_submission_status.__code__
     assert job_3._subscribers[1].__code__ == _Orchestrator._on_status_change.__code__
 
     # Same submit_id
     assert job_1.submit_id == job_2.submit_id == job_2bis.submit_id == job_3.submit_id
 
     # submission is created and correct
     assert len(_SubmissionManagerFactory._build_manager()._get_all()) == 1
-    submission = _SubmissionManagerFactory._build_manager()._get(job_1.submit_id)
     assert submission.submission_status == SubmissionStatus.BLOCKED
     assert submission.jobs == jobs
     assert submission.creation_date == s_time
     assert submission.entity_id == scenario.id
     assert submission.entity_type == "SCENARIO"
     assert submission.entity_config_id == "scenario_cfg"
+    assert submission.properties == {"no_of_retry": 10, "log": True, "log_file": "file_path"}
 
```

Comment:
Since you get the submission directly from the submit method, you don't get it from the Manager anymore. That means you don't test it is stored correctly.

```suggestion
# submission is stored
    submission = _SubmissionManagerFactory._build_manager()._get(job_1.submit_id)
    assert submission.submission_status == SubmissionStatus.BLOCKED
    assert submission.jobs == jobs
    assert submission.creation_date == s_time
    assert submission.entity_id == scenario.id
    assert submission.entity_type == "SCENARIO"
    assert submission.entity_config_id == "scenario_cfg"
    assert submission.properties == {"no_of_retry": 10, "log": True, "log_file": "file_path"}
```

What do you think? Maybe it could be a different test function.

Reply:
make perfect sense to me 😃

Reply:
added retrieving submission from repo before asserting